app/helpers/filters_helper.rb

```diff
@@ -5,4 +5,8 @@ def filter_url_query
       metric: { period: params&.dig(:metric, :period) }
     }
   end
+
+  def current_department
+    Department.find_by(name: params[:department_name])
```

Comment:
should it raise an exception if the department isn't found?

Reply:
i'm not sure, the `current_department` is only used under department views where department_name is always present, if this parameter is not present the url would crash. what do you think? 🤔

Reply:
so I guess we should use
```suggestion
    Department.find_by!(name: params[:department_name])
```